tests/models/test_model.py

```diff
@@ -227,6 +227,16 @@ def test_model_info():
         assert model_info_fetched.mlflow_version == loaded_model.mlflow_version
 
 
+def test_model_info_with_model_version():
+    with TempDir(chdr=True):
```

Comment:
why do we need to use a temporary directory and change the working directory?

Reply:
keep consistent with other tests in the suite, these suites all use `test` experiment, to avoid they influence each other.

Reply:
Can we call `mlflow.create_experiment` with `artifact_location=tmp_path`?